### PR TITLE
user: Show highlighted custom profile fields first

### PR DESCRIPTION
**Pull request by gnprice**
Draft PR because the API isn't yet final. This uses the API from the current server-side prototype, https://github.com/zulip/zulip/pull/21588 .

Fixes: #5363

### CODE REVIEW

**Review by chrisbobbe — COMMENTED** [review]
Thanks! LGTM, with one nit below; other than that, I guess we wait for the server PR to be merged.

File: src/api/modelTypes.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ export type CustomProfileField = {|
    * it's a string, one serializing a JSON object.
    */
   +field_data: string,
+  // TODO(server-6.0): New in FL ???.  When absent, treat as false.
+  // TODO(server): Use the final name for this property.
+  display_in_profile_popover?: boolean,
```
[the remark this answers is not in the document]
Comment (gnprice) [reply src/api/modelTypes.js]: Thanks! Fixed.

**Comment by alya** [comment]
The server PR has been merged.

**Comment by gnprice** [comment]
Updated, and tested manually against CZO. Merging!

**Pull request merged**